openlibrary/templates/type/author/view.html

```diff
@@ -179,6 +179,10 @@ <h3>$:_('Links <span class="gray small sansserif">(outside Open Library)')</span
                         <li>VIAF: <a itemprop="sameAs" href="https://viaf.org/viaf/$viaf">$viaf</a></li>
                     $if page.remote_ids.wikidata:
                         <li>Wikidata: <a itemprop="sameAs" href="https://tools.wmflabs.org/reasonator/?q=$page.remote_ids.wikidata">$page.remote_ids.wikidata</a></li>
+                    $if page.remote_ids.project_gutenberg:
```

Comment:
Ah, https://openlibrary.org/config/author contains things like label/link format. We want to replace this with a call to `get_author_config()`, and then walk through all the `identifiers`.

Reply:
Editions do something similar, so looking there might help?

Reply:
That's a great approach!  Should the existing links also be replaced?  Also, could you provide a sample prod URL configuration if they are different from the configs in #5093 (`@@@` used in place of the ID)?

Reply:
Can you see the page here: https://openlibrary.org/config/author ? It should be publicly visible. That's the prod config (Yep! @@@ is used). And yep! We should replace the old ones as well :0

Reply:
> Can you see the page here: https://openlibrary.org/config/author ?

Yes, I can see it.  Not sure why I didn't check first before asking...  Thanks!

Reply:
:D

Reply:
Updated.  New screenshot:

![remote_ids_updated](https://user-images.githubusercontent.com/28732543/123706884-de0a9080-d836-11eb-809b-d2d26add7766.png)